src/server_manager/web_app/gcp_account.ts

```diff
@@ -63,31 +64,40 @@ export class GcpAccount implements gcp.Account {
   async createServer(projectId: string, name: string, zone: gcp.Zone):
       Promise<server.ManagedServer> {
     const zoneLocator = {projectId, zoneId: zone.id};
-    const instance = await this.createInstance(zoneLocator, name);
-    const id = `${this.id}:${instance.id}`;
-    return new GcpServer(id, instance, this.apiClient);
+    const instanceName = makeGcpInstanceName();
+    const {instanceId, instanceCreation} =
+        await this.createInstance(zoneLocator, name, instanceName);
```

Comment:
Could you inline the createInstance function here?

It's not called anywhere else, and the duplicate name keeps confusing me.

Reply:
Done